plugins/modules/group.py

```diff
@@ -399,15 +442,14 @@ def main():
         else:
             result['msg'] = f"Group { name } already exists."
 
-    elif module.params['state'] == 'modify':
-        if not module.params['group_attributes'] and not module.params['user_list_action']:
-            result['msg'] = f"State is { state }. Please provide attributes or action."
-        else:
-            if group_exists(module):
-                result['msg'] = modify_group(module)
+            if not module.params['group_attributes'] and not module.params['user_list_action']:
+                result['msg'] = f"State is { state }. Please provide attributes or action."
             else:
-                result['msg'] = f"No group found in the system to modify the attributes: { name }"
-                module.fail_json(**result)
+                if group_exists(module):
+                    result['msg'] = modify_group(module)
+                else:
+                    result['msg'] = f"No group found in the system to modify the attributes: { name }"
```

Comment:
This case should not occur as we are already checking whether the group exists (through "else" in line number 442)